IntegrationTests/common/script/CompareMemPrintsFW.py

```diff
@@ -50,6 +50,10 @@ class ReferenceType(Enum):
     CM     = 'CandidateMatches'
     FM     = 'FullMatches'
     VMPROJ = 'VMProjections'
+    VMSME  = 'VMStubs'
+    VMSTE  = 'VMStubs'
+    AS     = 'AllStubs'
+    IL     = 'InputStubs'
```

Comment:
If easy, please add lines for all the memories in the whole L1Trk chain here, so script will work for TE-TC etc.

Reply:
OK!